examples/worlds/mecanum_drive.sdf

```diff
@@ -128,15 +128,16 @@
           </inertia>
         </inertial>
         <visual name='visual'>
+          <pose>0 0 0 1.5707 0 0</pose>
           <geometry>
-            <sphere>
+            <!--sphere>
```

Comment:
remove spheres and add a comment about the size of the wheels?

Reply:
https://github.com/gazebosim/gz-sim/pull/2250/commits/a7bb50b39b57268abe2c799bc1dd93ea2390935b